ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/AbstractCreateStreamCommand.java

```diff
@@ -159,11 +159,9 @@ private Schema getKsqlType(final String sqlType) {
 
   private Schema getKsqlComplexType(final String sqlType) {
     if (sqlType.startsWith("ARRAY")) {
-      return SchemaBuilder
-          .array(getKsqlType(sqlType.substring("ARRAY".length() + 1, sqlType.length() - 1)));
+      return SchemaBuilder.array(getKsqlType(sqlType.substring("ARRAY".length() + 1, sqlType.length() - 1)));
```

Comment:
perhaps we should turn "ARRAY" and "MAP" into constants?

Reply:
done